### PR TITLE
Added Swag Opportunity for Trigger.dev

### PR DESCRIPTION
**Pull request by BandhiyaHardik**
Added Swag Opportunity for Trigger.dev

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

This fixes https://github.com/swapagarwal/swag-for-dev/issues/1155

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -872,6 +872,15 @@
...
     "tags": ["stickers", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Trigger.dev",
+    "difficulty": "medium",
+    "description": "Submit PRs in trigger.dev or job-showcase repos and earn points to redeem for swag.",
+    "reference": "https://trigger.dev/hacktoberfest",
+    "image": "https://i.imgur.com/kelNqon.jpg",
+    "dateAdded": "2023-10-21T23:15:00.000Z",
+    "tags": ["clothing", "hacktoberfest"]
```
Comment (aslafy-z) [inline data.json:882]: Please list the other offerings using existing tags if possible.

**Review by aslafy-z — APPROVED** [review]

**Review by aslafy-z — CHANGES_REQUESTED** [review]
Please see my comment

File: data.json (hunk 1 above)
Comment (BandhiyaHardik) [reply data.json:882]: sure i will do it now thank you for letting me know

**Comment by BandhiyaHardik** [comment]
@aslafy-z  i have added tags of swags in it as per suggested

**Review by aslafy-z — APPROVED** [review]

**Pull request merged**